pandas/core/groupby/groupby.py

```diff
@@ -1097,7 +1097,8 @@ def count(self):
         # defined here for API doc
         raise NotImplementedError
 
-    @Substitution(name='groupby', see_also=_common_see_also)
+    @Substitution(name='groupby')
+    @Substitution(see_also=_common_see_also)
```

Comment:
If done there are also `%name` occurences in the `_common_see_also` that need to be substituted, so they cannot be done in one go.